package.json

```diff
@@ -30,15 +31,16 @@
     "server": "nodemon server",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "dev": "npm run start"
   },
   "devDependencies": {
     "path": "^0.12.7",
     "react-dev-utils": "^3.1.1",
     "react-scripts": "0.9.5"
   },
   "engines": {
-    "node": "8.x"
+    "node": "^10.3.0"
```

Comment:
Important to do in this particular PR? If so, I'll probably go through a full upgrade _before_ merging this, just to get everything up to date.

Reply:
I couldn't get it to run locally w/o upgrading it.

Saves me having to remember adding the line if it's not, dare I say, a big thing to do a full upgrade… :) but if it is, I'll just not commit it.